"Test has no assertions" warnings printed during test run

```
Test Suites: 106 passed, 106 total
Tests:       6 skipped, 1477 passed, 1483 total
Snapshots:   299 passed, 299 total
Time:        16.371 s
Ran all test suites.

/Users/mstange/code/perf.html/src/test/components/StackChart.test.js
  240:3  warning  Test has no assertions  jest/expect-expect
  244:3  warning  Test has no assertions  jest/expect-expect

/Users/mstange/code/perf.html/src/test/components/TrackContextMenu.test.js
  447:5  warning  Test has no assertions  jest/expect-expect
  559:5  warning  Test has no assertions  jest/expect-expect

✖ 4 problems (0 errors, 4 warnings)
```

--- a/src/test/components/StackChart.test.js
+++ b/src/test/components/StackChart.test.js
@@ -236,12 +236,10 @@ describe('MarkerChart', function () {
   });
 
   // TODO implement selecting user timing markers #2355
-  // eslint-disable-next-line jest/no-disabled-tests
-  it.skip('can select a marker when clicking the chart', function () {});
+  it.todo('can select a marker when clicking the chart');
 
   // TODO implement selecting user timing markers #2355
-  // eslint-disable-next-line jest/no-disabled-tests
-  it.skip('can right click a marker and show a context menu', function () {});
+  it.todo('can right click a marker and show a context menu');
 
   it('shows a tooltip when hovering', () => {
     const { getTooltip, moveMouse, findFillTextPosition } = setupUserTimings({

--- a/src/test/components/TrackContextMenu.test.js
+++ b/src/test/components/TrackContextMenu.test.js
@@ -443,10 +443,8 @@ describe('timeline/TrackContextMenu', function () {
       expect(getHiddenGlobalTracks(getState()).has(trackIndex)).toBe(false);
     });
 
-    // eslint-disable-next-line jest/no-disabled-tests
-    it.skip('can present a disabled isolate item on non-process tracks', function () {
-      // TODO - We should wait until we have some real tracks without a thread index.
-    });
+    // TODO - We should wait until we have some real tracks without a thread index.
+    it.todo('can present a disabled isolate item on non-process tracks');
 
     it('network track will be displayed when a number is not set for ctxId', () => {
       const { container } = setupGlobalTrack(getNetworkTrackProfile(), 0);
@@ -555,10 +553,10 @@ describe('timeline/TrackContextMenu', function () {
       expect(getHiddenLocalTracks(getState(), pid).has(trackIndex)).toBe(false);
     });
 
-    // eslint-disable-next-line jest/no-disabled-tests
-    it.skip('can isolate a non-thread track, as long as there process has a thread index', function () {
-      // TODO - We should wait until we have some real non-thread tracks
-    });
+    // TODO - We should wait until we have some real non-thread tracks
+    it.todo(
+      'can isolate a non-thread track, as long as there process has a thread index'
+    );
   });
 
   describe('global / local track visibility interplay', function () {

